lib/widgets/read_times_tamp.dart

```diff
@@ -1,92 +1,15 @@
-import 'dart:ui';
-
-import 'package:intl/intl.dart';
 import 'package:timeago/timeago.dart' as timeago;
 
 String readTimestamp(String timestamp) {
```

Comment:
name this function according to what its doing. 

And leave comments. and comment example of what it returns

Reply:
variables name where fix and I added a comment with a link to timeago API for references.